src/table/index.js

```diff
@@ -158,29 +133,32 @@ class TableField extends Component {
     this.props.onChange(removePosition(filteredRows));
   }
   handleRowSelect(row, isSelected, e) {
+    const { onChange } = this.props;
+
     const {
-      data,
+      data = [],
       selectRow: { onSelectRow: { fieldToUpdate = "picked" } }
     } = this.tableConf;
-    let filteredRows = (data || []).map(item => {
```

Comment:
Just wanted to double check that this wouldn't break any existing feature

Reply:
Confirmed that the built-in select all checkbox still works using the medications form